docs/usage/kafka_containers.md

```diff
@@ -0,0 +1,48 @@
+# Kafka Containers
+
+Test Containers can be used to automatically instantiate and manage [Kafka](https://kafka.apache.org) containers.
+More precisely Test Containers use official Docker images for [Confluent OSS Platform](https://hub.docker.com/r/confluentinc/cp-kafka/)
+
+## Benefits
+
+* Running single node Kafka installation with just one line of code
+* No need to manage external Zookeeper installation, required by Kafka. But see [below](#zookeeper)
+
+## Example
+
+The following field in your JUnit UI test class will prepare a container running Kafka:
+```java
+@Rule
+public KafkaContainer kafka = new KafkaContainer();
+```
+        
+Now your tests or any other process running on your machine can get access to running Kafka broker by using the following bootstrap server location:
+```java
+kafka.getBootstrapServers()
+```
+
+If your test needs to run some other docker container which needs access to the Kafka, do the following:
```

Comment:
Could you please move this section to the end of this document? So that only interested parties will read it :D